### DISCUSSION TITLE
Brew HTTPD 2.4.46 sophos error

### DISCUSSION THREAD

**Discussion by joshyhud** [Homebrew/discussions#132] [Everyday usage]
Brew version of HTTPD 2.4.46 is showing in my antivirus software (using sophos) as ransomware and blocking the use of the programme.

This might be something that needs to be looked into or is there an update I have missed? I can permit the running of the application through the sophos dashboard but thought I'd raise it.

**Comment by SMillerDev** [comment]
We don't have any reason to believe it's ransomware, sounds like a false positive to me. You're the only one of [150k users](https://formulae.brew.sh/formula/httpd) that thinks it is.
